projects/media-viewer/src/lib/store/reducers/annotatons.reducer.ts

```diff
@@ -11,6 +11,7 @@ export interface AnnotationSetState {
   commentEntities: {[id: string]: Comment} | {};
   selectedAnnotation: SelectionAnnotation;
   commentSearchQueries: {commentSearch: string;};
+  commentSummaryFilers: {hasFilter: boolean; filters: any};
```

Comment:
I assume this should say commentSummaryFilters

Reply:
hahah

Reply:
done